jonelle-ramlal/07/exercise.test.js

```diff
@@ -0,0 +1,34 @@
+describe('07', () => {
+    test('Using the "forEach" array method, print the current element', () => {
+        const spyOnLog = vi.spyOn(console, 'log');
+        /**
+         * You are expected to use the forEach array method
+         * 
+         * You are expected to use the console.log() method
+         * 
+         * You are expected to use the template literal syntax
+         * Example: `The current element is ....`
+         */
+        const array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+        // Your code here
+
+        array.forEach(printElement)
+
+        function printElement(item){
+            console.log("The current element is " + item)
```

Comment:
Expected you to use template literal syntax instead of quotes
```suggestion
            console.log(`The current element is ${item}`)
```